### ISSUE TITLE
Pre-built hobbyist image won't flash to sdcard

### ISSUE THREAD

**Issue by johncblacker** [PyImageSearch/Raspberry-Pi-for-Computer-Vision-RPi4CV#5]
_Please use this template while reporting bugs inside RPi4CV. Issues related to personal project questions or code that we haven't authored will automatically be closed._

## Are you using one of our development environments? 
Yes, I'm trying to flash the hobbyist image downloaded when I purchased the bundle.
We provide a preconfigured AWS AMI and a VM to help our customers skip the environment setup part and jump straight to development. We highly suggest using one of our development environments to run our code. 

If the issue is not related to code, then you can simply skip this question. 

## If you are using your own development environment then please provide the following details.
No, trying to flash the pi2/3/4 img to an sdcard
* GPU model and CUDA version: (skip if not application) 
* OS Platform: 
* Python version: 
* OpenCV version: 
* TensorFlow version:

## Describe the issue 
When I try flashing the image using balena etched, it reports during the decompress phase that the image may be
corrupt and therefore won't flash it.
When I try flashing the image using the RPi imager, it get down to nearly the end of the verify process then
reports that what was flashed doesn't match the image.  I tried booting the image anyway, but it will not complete
the boot process, reports problems with the image.
Please be clear and concise. Don't forget to include the Bundle and chapter names where you have found issues. 

## Any other information
I have tried the flash steps with this image from two different computers, two different sdcards, and I've downloaded the
image zip several times.  All attempts result in the same errors as noted above.  I did finally have to download the official RPi Buster image and go through the process of setting up the environment the quick and easy way (not building opencv from sources) but I do want to use a pre-built image with the complete OpenCV.  Either the zip is corrupted or the image(s) are corrupt.  Perhaps you can make available just the RPi 2/3/4 pre-built image in uncompressed format?

**Comment by johncblacker** [comment]
I noted that the website for installing says this image can only be flashed to a 32GB card (seems strange to me, but OK) so
I got a 32GB card and tried again.  I still get same error:  with Balena Etcher a window opens that says:
Attention
Something wen wrong.  If it is a compress image, please check that the archive is not corrupted

incorrect data check
(Cancel)                                                   (Retry)

Retry doesn't work.

**Comment by vikaskookna** [comment]
@johncblacker Thanks for reaching out to us can you confirm that you're using this [blog](https://www.pyimagesearch.com/2016/11/21/raspbian-opencv-pre-configured-and-pre-installed/?_ga=2.242652290.1878012741.1643646724-1614231343.1634395940) for setting up your image, and you're not using M1 mac, our other customers are able to successfully set up the image and had no complaints.  While we have users using M1 mac reporting problems we're right now in the process of finding solutions.

**Comment by johncblacker** [comment]
I am using the instructions in that blog.  Again, I've tried Balena Etcher and the RPi Imager, neither will work.  I'm attempting to flash from a Windows 10 computer.  As of right now, I'm trying to use Express zip to unzip the pi 3/4 gz file to get to a 
.img file and it's been running over an hour...I don't think it's working...but I'll let it go for a while.  It shouldn't be taking this long to unzip the file.  And, yes, I've tried using a 32GB sdcard.
